api-service/src/routes/Router.ts

```diff
@@ -54,6 +55,8 @@ router.post("/datasets/copy", setDataToRequestObject("api.datasets.copy"), onReq
 router.post("/connectors/list", setDataToRequestObject("api.connectors.list"), onRequest({ entity: Entity.Management }), ConnectorsList);
 router.get("/connectors/read/:id", setDataToRequestObject("api.connectors.read"), onRequest({entity: Entity.Management }), ConnectorsRead);
 router.post("/datasets/import", setDataToRequestObject("api.datasets.import"), onRequest({ entity: Entity.Management }), DatasetImport);
+router.get("/datasets/generate-fields/:dataset_id", setDataToRequestObject("api.generate.fields"), onRequest({ entity: Entity.Management }), generateFields);
```

Comment:
shouldn't this be a webconsole API? Move it to webconsole with the path as /webconsole/dataset/flatten/:datasetId

Reply:
done